apps/src/templates/sectionProgress/progressTables/progressTableStyles.scss

```diff
@@ -39,12 +31,12 @@ $content-view-width: $content-width - $student-list-width;
   }
   .primary-row {
```

Comment:
this file can't be a module file because some classes (like `primary-row`) are implemented by external libraries. in this case, we have a library that renders the progress table and applies its own classes